changes/views.py

```diff
@@ -72,3 +78,54 @@ def post(self, request, *args, **kwargs):
                               action='momconnect_nonloss_optout',
                               data={'reason': 'sms_failure'})
         return Response(status=status.HTTP_201_CREATED)
+
+
+class ReceiveAdminOptout(mixins.CreateModelMixin,
```

Comment:
We don't need the `CreateModelMixing` if we're not using the `.create` function.